src/components/ArticleCard/ArticleCard.jsx

```diff
@@ -0,0 +1,16 @@
+const ArticleCard = ({article}) => {
+  return (
+    <article>
+          <img src={article.article_img_url} />
+          <h2> {article.title}</h2>
+          <h3>
+            Created by {article.author} on {article.created_at.slice(0, 10)}
```

Comment:
Looks like this would be an excellent opportunity for a (fully tested!) formatDate function...